test/aliases.js

```diff
@@ -0,0 +1,59 @@
+import { join } from 'path'
+import { version } from 'process'
+
+import test from 'ava'
+
+import { resolveVersionRangeAlias } from '../src/aliases.js'
```

Comment:
Would it be possible to import the entry point `../src/main.js` instead? Integration testing would be more robust to future changes and ensure the logic works for the end user.

Reply:
Great! Would it be possible to replace `../src/aliases.js` with `../src/main.js`. Integration tests would be better than unit tests in this instance IMHO.